README.md

```diff
@@ -84,18 +84,20 @@ bin/rails g
 
 ## Development
 
-We use the 2 embeded Rails apps [Rails 5 Example](./recreate_rails5_example) and [Rails 6 Example](./recreate_rails6_example) to test the usage of the generators.
+Install the versions of Node and Ruby specified in `.node-version` and `.ruby-version` on your machine. https://asdf-vm.com/ is a great tool for managing language versions. Then run `npm install -g yarn`.
 
-`recreate_rails6_example` is simply a fresh generated Rails 6 app. To recreate it use:
+We use the embeded Rails apps [Rails 7 Example](./recreate_rails7_example) to test the usage of the generators.
+
+`recreate_rails7_example` is simply a freshly generated Rails 7 app. To recreate it use:
 
 ```shell
-bin/recreate_rails6_example
+bin/recreate_rails7_example
```

Comment:
The results of this command should be committed to git so we can see how the default Rails 7 files change from version to version. Right now when I run this generator, it changes a lot of the default files we already have in git which makes it hard to see what the generators are actually modifying.